cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -0,0 +1,288 @@
+                              <li class="list_item"><a class="list_link" href="#">Compliance guides</a></li>
+                              <li class="list_item"><a class="list_link" href="#">Mortgage</a></li>
+                              <li class="list_item"><a class="list_link" href="#">College loans</a></li>
+                          </ul>
+                      </div>
+                  </form>
+              </div>
+          </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-global-search
+            button.m-global-search_trigger
+              span.cf-icon
+              span.m-global-search_trigger-label
+                span.u-visually-hidden
+            .m-global-search_content
+              form.m-global-search_content-form
+                […
+                .input-with-btn
+                .input-contains-label
+                from cf-forms
+                …]
+                .m-global-search_content-suggestions
+                  .h5
+                  […
+                  .list.list__horizontal
+                  from cf-typography
+                  …]
+  tags:
+    - cfgov-molecules
+*/
+
+.m-global-search {
+
+    // Absolute position when there is a flyout at mobile/tablet sizes.
+    .respond-to-max( @bp-sm-max, {
+        // Make the height large so that the overflowing flyout can be hidden,
+        // but not get cropped when it's visible.
+        height: 500px;
+        width: 100%;
+        position: absolute;
+        top: 0;
+        overflow-x: hidden;
+        // TODO: This will be ignored in IE8-10,
+        //       leading to the overflow preventing clicks on the links.
+        pointer-events: none;
+    } );
+
+    &_trigger {
+        height: unit( 45px / 18px, em );
+        min-width: unit( 45px / 18px, em );
+        padding-left: unit( @grid_gutter-width / 2 / 18px, em );
+        padding-right: unit( @grid_gutter-width / 2 / 18px, em );
+
+        float: right;
+        position: relative;
+        z-index: 20;
+
+        background: @white;
+        border: none;
+        color: @black;
+        font-size: unit( 18px / @base-font-size-px, em );
+        pointer-events: auto;
+
+        &:focus {
+            outline: 1px dotted @black;
+        }
+
+        .cf-icon:before {
+            content: @cf-icon-search;
+            font-size: unit( 20px / 18px, em );
+        }
+
+        // Show "Search" text in trigger at tablet sizes.
+        .respond-to-min( @bp-sm-min {
+            &-label:before {
+                .webfont-medium();
+                content: 'Search';
+            }
+        } );
+
+        // Only show flyout and close trigger at tablet/mobile sizes.
+        .respond-to-max( @bp-sm-max, {
+            height: unit( 60px / 18px, em );
+            min-width: unit( 60px / 18px, em );
+            padding-top: @margin_half__em;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/92570e377562e0b76c846ad23be7b7b18aea5b59 `unit( @grid_gutter-width / 2 / 18px, em );`